docs/gettinginvolved/infrastructure/release-process.rst

```diff
@@ -118,7 +123,8 @@ Preparing For a Release
 
    - The NEWS date format is YYYY-MM-DD.
    - The NEWS file should also be updated for each pre-release and Weekly Development Release, however there should be only one NEWS entry for each Major Marketing Release and Minor Marketing Release.
-   - This means that in doing a release, you may have to change the NEWS heading from a previous Weekly Development Release or pre-release.
+     This means that in doing a release, you may have to remove the previous development release or pre-release header, merging the changes from that previous release into the current release.
+     This applies to both Weekly Development Releases or pre-releases.
```

Comment:
"This applies to both Weekly Development Releases or pre-releases." is redundant.